packages/metrics/tests/e2e/decorator.test.ts

```diff
@@ -98,24 +100,19 @@ describe('happy cases', () => {
       .promise();
     expect(coldStartMetrics.Metrics?.length).toBe(1);
     const coldStartMetric = coldStartMetrics.Metrics?.[0];
-    expect(coldStartMetric?.Dimensions).toStrictEqual([
-      { Name: 'service', Value: expectedServiceName },
-      { Name: 'function_name', Value: functionName },
-      { Name: Object.keys(expectedDefaultDimensions)[0], Value: expectedDefaultDimensions.MyDimension },
-    ]);
+    expect(coldStartMetric?.Dimensions).toStrictEqual(expectedDimensions);
 
     // Check coldstart metric value
+    const adjustedStartTime = new Date(startTime.getTime() - 60 * 1000);
+    const endTime = new Date(new Date().getTime() + 60 * 1000);
+    console.log(`Manual command: aws cloudwatch get-metric-statistics --namespace ${expectedNamespace} --metric-name ColdStart --start-time ${Math.floor(adjustedStartTime.getTime()/1000)} --end-time ${Math.floor(endTime.getTime()/1000)} --statistics 'Sum' --period 60 --dimensions '${JSON.stringify(expectedDimensions)}'`);
```

Comment:
I like this a lot, will do it also in the Tracer tests next time I get around it

Reply:
I did it for tracer as well , I can add it to the PR if you want

Reply:
If you have them already yes, otherwise I'll do them at some point. I think they are useful for troubleshooting esp for Tracer

Reply:
added